### ISSUE TITLE
Can't open any cosmetics.

### ISSUE THREAD

**Issue by Kinokoda** [hisprofile/TF2-Trifecta#2]
Heyo, this project fuckin rocks, but I have a feeling I didn't set something up properly, and I am kinda stuck when it comes to troubleshooting.
![image](https://user-images.githubusercontent.com/56170513/233737124-afa0d0cf-1438-44fb-add2-5e69ac42afe9.png)
I have been getting this error whenever I try to deploy a cosmetic.

Any idea what could be causing this? Is there any more info I could bring here?

Sorry to bother, have a good day.

**Comment by Kinokoda** [comment]
![image](https://user-images.githubusercontent.com/56170513/233804667-61a56644-1330-4021-b896-c8bfff5c88c9.png)

**Comment by hisprofile** [comment]
Sorry for the late reply. For the first screenshot, the error means that the object that was supposed to be spawned was not found. This may be due to a corrupted .blend file. I know this *could* happen when downloading the entire TF2 Collection as a .zip file, instead of manually downloading each file individually. Alternatively, if you have entries setup for each required asset path, you can just download each asset through the updater panel in Scene Properties.

As for the second screenshot, this is caused by a duplicate class being created. Registering two classes will work just fine, as you are essentially just overwriting the original with a duplicate, but trying to unregister one twice is what causes the issue. This issue was fixed in the latest update. Apologies for it saying that it is still 2.0.4, I forgot to move it up to 2.0.5.

**Comment by Kinokoda** [comment]
Please do not apologize, this isnt any urgent manner. 

As for the error I will try reinstalling the TF2 Blender Port cosmetics.

Thank you a LOT for your work. Blender is much more fun than SFM, and I would have never tried it out had it not been for this project. (and splatoon 3 lol)

Have a great rest of your day.

**Comment by Kinokoda** [comment]
After reinstalling all the files and setting up the paths again, the IK rigs work fine and actually rotate. But the cosmetic error persists.
![image](https://user-images.githubusercontent.com/56170513/233836454-2f3ef7de-74e7-458a-a678-13261144c7c4.png)
![image](https://user-images.githubusercontent.com/56170513/233836486-d341be1a-e996-4d45-a214-263421138a9c.png)
![image](https://user-images.githubusercontent.com/56170513/233836591-6f7a5052-320e-46af-be48-5019584c7f5d.png)
![image](https://user-images.githubusercontent.com/56170513/233836613-3766e9b9-73d6-4d75-ac92-9f718b82a74e.png)


Have I done something wrong?

**Comment by Kinokoda** [comment]
![image](https://user-images.githubusercontent.com/56170513/233837034-ef8db1e9-b2d8-4754-8bf7-575d13db79f6.png)

If I dont figure this problem out, Im probably just gonna find the cosmetics manually, no biggie.

**Comment by hisprofile** [comment]
Make sure your TF2-V3 entry is set as a folder, do not add entries for each file inside of TF2-V3. It may not be working due to the entries overlapping their names, like "engineer" may be pointing to TF2-V3\engineer.blend, not engineercosmetics.blend.

I don't mind continuing this on discord, but it would be good to keep this visible so that others may find the solution through this.

**Comment by Kinokoda** [comment]
Ok I have fixed ze problem. Here are the changes I made:
![image](https://user-images.githubusercontent.com/56170513/233863420-9e642787-3360-4235-8172-0455d5458175.png)
![image](https://user-images.githubusercontent.com/56170513/233863446-ce2f165b-af7e-4f33-83e5-2347e4dfe4a5.png)
![image](https://user-images.githubusercontent.com/56170513/233863468-df27f094-3313-46de-879a-39469f3ecdaf.png)

I already have set my asset library to the same drive "The TF2 Collection" is in so I guess that was what was over lapping. I also added entries for the cosmetics in the trifecta plugin and I changed the name of  the entries so that it matched what the plugin needed. And now it works!

Thanks a TON for your help, wouldn't have made it otherwise.
